src/pages/api/graphql/schema.graphql

```diff
@@ -0,0 +1,26 @@
+scalar UUID
+
+type Column {
```

Comment:
Move this file away from api folder. Everything inside that can be called by a client.